fury/actor.py

```diff
@@ -3443,6 +3443,20 @@ def texture_update(texture_actor, arr):
 
 
 def _textured_sphere_source(theta=60, phi=60):
+    """
+    Uses vtkTexturedSphereSource to set the theta and phi
```

Comment:
- `Use` instead of `Uses`   -> Imperative mood everywhere.
- Dot at the end of the sentence
- Also, it should be a one-liner. See numpy standard that we follow: https://numpydoc.readthedocs.io/en/latest/format.html